erp-client/src/styles.scss

```diff
@@ -1,2 +1,69 @@
-/* You can add global styles to this file, and also import other style files */
-@import '@angular/material/prebuilt-themes/indigo-pink.css';
\ No newline at end of file
+// Angular Material Core Theme
+@import '~@angular/material/theming';
+
+// Include non-theme styles for core.
+@include mat-core();
+
+// Define your application's custom theme.
+// TODO: Determine what actual colors we want to use.
+$primary: mat-palette($mat-indigo);
+$accent:  mat-palette($mat-pink, A200, A100, A400);
+$theme: mat-light-theme($primary, $accent);
+
+// Include theme styles for Angular Material components.
+@include angular-material-theme($theme);
+
+//
+// dom elements
+//
+
+body {
+  background: #fafafa;
+  margin: 0;
+  padding: 0;
+}
+
+
+//
+// container
+//
+
+.container {
+  width: 80%;
+  max-width: 1200px;
+  margin: 60px auto;
+}
+
+
+//
+// toolbar
+//
+
+mat-toolbar.mat-toolbar {
+  background-color: mat-color($primary, 500);
+  color: #fff;
+}
+
+
+//
+// snackbar
+//
+
+.mat-simple-snackbar-action {
+  color: #fff;
+}
+
+.snackbar-success {
+  background-color: mat-color($mat-green, 800);
+  color: mat-color($mat-green, 50);
+}
+
+.snackbar-error {
+  background-color: mat-color($mat-red, 800);
+  color: mat-color($mat-red, 50);
+}
+
+.snackbar-warning {
+  background-color: mat-color($mat-orange, 800);
+  color: mat-color($mat-orange, 50);
+}
```

Comment:
Added these styles here instead of at the component level, bc I thought maybe we'd want to use them in other areas. These could be split into separate files and imported here if we'd like to clean it up a bit.

Reply:
Nice 👌